aws.yaml

```diff
@@ -1,5 +1,63 @@
 apiVersion: release.giantswarm.io/v1alpha1
 kind: Release
+metadata:
+  annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
+  name: v11.3.0
+spec:
+  state: active
+  date: 2020-05-05T15:00:00Z
+  apps:
+  - name: cert-exporter
+    version: 1.2.1
+  - name: cert-manager
+    componentVersion: 0.9.0
+    version: 1.0.7
+  - name: chart-operator
+    version: 0.12.4
+  - name: cluster-autoscaler
+    componentVersion: 1.16.2
+    version: 1.1.4
+  - name: coredns
+    componentVersion: 1.6.5
+    version: 1.1.8
+  - name: external-dns
+    componentVersion: 0.5.18
+    version: 1.2.0
+  - name: kiam
+    componentVersion: 3.5.0
+    version: 1.2.2
+  - name: kube-state-metrics
+    componentVersion: 1.9.5
+    version: 1.0.5
+  - name: metrics-server
+    componentVersion: 0.3.3
+    version: 1.0.0
+  - name: net-exporter
+    version: 1.7.0
+  - name: node-exporter
+    componentVersion: 0.18.1
+    version: 1.2.0
+  components:
+  - name: app-operator
+    version: 1.0.0
+  - name: aws-operator
+    version: 8.4.1-dev
+  - name: cert-operator
+    version: 0.1.0
+  - name: cluster-operator
+    version: 2.1.10
+  - name: kubernetes
+    version: 1.16.9
+  - name: containerlinux
+    version: 2345.3.1
+  - name: calico
+    version: 3.10.1
+  - name: etcd
+    version: 3.3.17
```

Comment:
Please also add component `name: aws-cni` with `version: 1.6.0`. We just added that in retrospect for informational purposes to the releases making use of it.

Reply:
Done!